lib/manager/pip_requirements/extract.ts

```diff
@@ -11,6 +12,9 @@ import type { ExtractConfig, PackageDependency, PackageFile } from '../types';
 export const packagePattern =
   '[a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9._-]*[a-zA-Z0-9]';
 const extrasPattern = '(?:\\s*\\[[^\\]]+\\])?';
+const packageGitRegex = regEx(
+  /(?<source>(?:git\+)(git|ssh|https):\/\/(?<gitUrl>(?<user>.*)@(?<hostname>[\w.-]+)(?<delimiter>\/)(?<scmPath>.*\/(?<depName>[\w/-]+))(\.git)?(?:@(?<version>.*))))/
```

Comment:
@rarkins Do we want/need to add a new section "pip" to the section `Package Manager Credentials for Artifact Updating` in the documentation, to explain that you can now use pip with private Git repositories?

1. Go to: https://docs.renovatebot.com/getting-started/private-packages/#package-manager-credentials-for-artifact-updating
1. Check the sidebar on the right, see items for `bundler`, `composer`, `gomod`, `helm`, `npm`, `nuget` and finally `poetry`
1. Maybe add new item `pip` here?

Reply:
I'm not sure we actually can. I believe the author's intent is to embed an ssh private key into the container/VM so that it's transparent to Renovate. This is not currently a solution I'm comfortable with making "official" because Eg it doesn't work with sidecar containers.

Reply:
> I'm not sure we actually can. I believe the author's intent is to embed an ssh private key into the container/VM so that it's transparent to Renovate. This is not currently a solution I'm comfortable with making "official" because Eg it doesn't work with sidecar containers.

Yes thats my intention.